Index.html

```diff
@@ -0,0 +1,303 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>TravelPro</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <section class="logo">
+          <img
+            src="images\logo.png"
+            alt="Pilania Travel Company Logo"
+            width="50"
+          />
+        </section>
+        <section class="nav-items">
+          <ul>
+            <li>Destinations</li>
+            <li>Hotels</li>
+            <li>Flights</li>
+            <li>Bookings</li>
+            <li>Login</li>
+            <div class="signup-btn">Sign up</div>
+            <div class="language-dropdown">
+              <span class="dropdown-trigger">EN&#9660;</span>
+              <ul class="dropdown-content">
+                <li>Hindi</li>
+                <li>German</li>
+              </ul>
+            </div>
+          </ul>
+        </section>
+      </nav>
+      <main>

```

Comment:
Main content shouldn't be placed inside `header` tag.
move `main` tag to outside. Place it after `header` tag

Reply:
resolved, thanks for the input.